docs/data/sql_functions.yml

```diff
@@ -1069,6 +1072,30 @@ json:
         )
       )
       ```
+  - sql: JSON(value)
```

Comment:
I do not see a function called JSON in the other SQL implementations . I do not think we should be adding a newly name function.
I see function JSON_VALUE - can we not use this instead?

Reply:
Hey, @davidradl. The SQL specification introduced a new JSON data type (ISO/IEC 9075-2:2023 introduced JSON type (T801)). There are some databases that already implemented it and this is the first simple initial support for a specific use case.

Reply:
I understand there is a new json type - please could you point me to databases that have a SQL function called JSON().

I look at [Oracle](https://docs.oracle.com/en/database/oracle/oracle-database/12.2/adjsn/generation.html#GUID-1084A518-A44A-4654-A796-C1DD4D8EC2AA) and see that they use json_object for the nested json_object.
Flink seems to have a different implementation of json_object so it cannot be used in this way.